example-wp4/src/index.js

```diff
@@ -1,6 +1,8 @@
 'use strict';
 
 require('./index.html');
-var Elm = require('./Main.elm');
+var elm = require('./Main.elm');
```

Comment:
Maybe `var Elm = require('./Main.elm').Elm` is better? (or `const { Elm } = ...`).

Reply:
Thanks, will do (I am no JS expert :-) ).